Purchases/Public/Error Handling/ErrorUtils.swift

```diff
@@ -48,14 +47,14 @@ import StoreKit
     }
 
     /**
-     * Maps an [BackendErrorCode] code to a [ErrorCode]. code. Constructs an Error with the mapped code and adds a
-     * [RCUnderlyingErrorKey] in the [NSError.userInfo] dictionary. The backend error code will be mapped using
-     * [BackendErrorCode.toPurchasesErrorCode()].
+     * Maps an `BackendErrorCode` code to a `ErrorCode`. code. Constructs an Error with the mapped code and adds a
+     * `RCUnderlyingErrorKey` in the `NSError.userInfo` dictionary. The backend error code will be mapped using
+     * `BackendErrorCode.toPurchasesErrorCode()`.
      *
-     * - Parameter: backendCode The numerical value of the error.
-     * - Parameter: backendMessage The message of the errror contained under the [NSUnderlyingErrorKey] key in the
+     * - Parameter backendCode: The numerical value of the error.
+     * - Parameter backendMessage: The message of the errror contained under the `NSUnderlyingErrorKey` key in the
      * UserInfo dictionary.
-     * - Parameter: finishable Will be added to the UserInfo dictionary under the [RCFinishableKey] to indicate if the
+     * - Parameter finishable: Will be added to the UserInfo dictionary under the `RCFinishableKey` to indicate if the
```

Comment:
this is now `finishableKey`, right?

Reply:
It's finishableKey on the error, but the public API for creating the error hasn't changed.